playground/backend/cmd/server/controller.go

```diff
@@ -241,22 +241,24 @@ func (controller *playgroundController) Cancel(ctx context.Context, info *pb.Can
 }
 
 // GetPrecompiledObjects returns the list of examples
+// - If SDK and category are unspecified in the request, gets the whole catalog from the cache
+//	- If there is no catalog in the cache, gets the catalog from the Storage and saves it to the cache
+// - If SDK or category is specified in the request, gets the specific catalog from the Storage
```

Comment:
It seems that the comment here doesn't match the method logic. The method gets a catalog from the cache in all cases, not only if `SDK and category are unspecified`

Reply:
Thanks for noticing, changed!